### PR TITLE
Add missing QLDoc for public declarations

### PR DESCRIPTION
**Pull request by owen-mc**

### CODE REVIEW

File: ql/src/semmle/go/frameworks/Beego.qll [hunk 1]
```diff
@@ -7,6 +7,10 @@ import go
 import semmle.go.security.Xss
 private import semmle.go.security.SafeUrlFlowCustomizations
 
+/**
+ * Provides classes for working with untrusted flow sources, sinks and taint propagators
+ * from the `github.com/beego/beego` package.
```
Comment (intrigus-lgtm) [inline ql/src/semmle/go/frameworks/Beego.qll:12]: Maybe convert the package reference to a link as is done below?

File: ql/src/semmle/go/frameworks/BeegoOrm.qll [hunk 2]
```diff
@@ -6,6 +6,10 @@
 import go
 private import semmle.go.security.StoredXssCustomizations
 
+/**
+ * Provides classes for working with untrusted flow sources, sinks and taint propagators
+ * from the `github.com/astaxie/beego/orm` subpackage.
```
Comment (intrigus-lgtm) [inline ql/src/semmle/go/frameworks/BeegoOrm.qll:11]: Dito here.

**Comment by owen-mc** [comment]
Thanks @intrigus-lgtm, I've made those changes.

File: ql/src/semmle/go/security/InsecureRandomnessCustomizations.qll [hunk 3]
```diff
@@ -32,6 +36,9 @@ module InsecureRandomness {
     InsecureRandomSource() { this.getTarget().getPackage().getPath() = "math/rand" }
   }
 
+  /**
+   * Gets an interface outside of crypto which is the same as an interface in crypto.
```
Comment (sauyon) [inline ql/src/semmle/go/security/InsecureRandomnessCustomizations.qll:40]: I would say "outside of the `crypto` package".

**Review by smowton — APPROVED** [review]

**Pull request merged**